common/main.c

```diff
@@ -23,6 +23,9 @@ main(void)
 
 	console_init(DEV_UART0);
 	dm_init();
+#if TEST
+	run_tests();
```

Comment:
You need to include `test.h` to call this function

Reply:
done